### ISSUE TITLE
Creating two databases broken in current vagrant development version

### ISSUE THREAD

**Issue by sneumann** [MassBank/MassBank-web#30]
How to reproduce:

1. Create a second database with mbadmin
1. Access http://192.168.XX.XX/MassBank/RecordIndex.html 

Observe the following stack trace:

```
message An exception occurred processing JSP page /jsp/RecordIndex.jsp at line 115

description The server encountered an internal error that prevented it from fulfilling this request.

exception

org.apache.jasper.JasperException: An exception occurred processing JSP page /jsp/RecordIndex.jsp at line 115

112: 			val = siteNameList[siteNo] + "\t" + siteNo;
113: 		}
114: 		else {
115: 			val = fields[0].split(":")[1];
116: 		}
117: 		int count = Integer.parseInt( fields[1] );
118: 		if ( key.equals("site") ) {			// Contributor


Stacktrace:
	org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:568)
	org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:470)
	org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:395)
	org.apache.jasper.servlet.JspServlet.service(JspServlet.java:339)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:727)

root cause

java.lang.ArrayIndexOutOfBoundsException: 1
	org.apache.jsp.jsp.RecordIndex_jsp._jspService(RecordIndex_jsp.java:318)
	org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:727)
	org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:432)
	org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:395)
	org.apache.jasper.servlet.JspServlet.service(JspServlet.java:339)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:727)
```

Deleting the second database and going back to record index "fixes" the problem.

**Comment by sneumann** [comment]
The error is here: https://github.com/MassBank/MassBank-web/blob/master/modules/tomcat/webapps/MassBank/jsp/RecordIndex.jsp#L115 
This is inside a loop iterating over a key-vale list inside `String line = (String)result.get(i);`
which is retrieved from `ArrayList result = mbcommon.execMultiDispatcher( serverUrl, typeName, "allcat=1" );` .
The `execMultiDispatcher` is defined in https://github.com/MassBank/MassBank-web/blob/6a4f6595e4658705624cf76e4e65312d7f945392/Tomcat/webapps/ROOT/WEB-INF/src/massbank/MassBankCommon.java#L146 and directly calls  `execDispatcher` in the same file https://github.com/MassBank/MassBank-web/blob/6a4f6595e4658705624cf76e4e65312d7f945392/Tomcat/webapps/ROOT/WEB-INF/src/massbank/MassBankCommon.java#L160, but I fail to check 
what results are being returned.

**Comment by tsufz** [comment]
Closed by #31
